src/api/index.js

```diff
@@ -153,13 +166,37 @@ export async function fetchUser(user, accessToken) {
 }
 
 export async function fetchUserOrgs(user, accessToken) {
-  const ORGS_ENDPOINT = `${root}/users/${user}/orgs`;
+  const getOrganizationsQuery = `
+    query($login: String!) {
+      user(login: $login) {
+          organizations(first: 100) {
+            edges {
+              node {
+                id
+                login
+                name
+                avatarUrl
```

Comment:
Seconding @machour's comment. I still don't know how exactly GraphQL works, but can we not stick to the same mention of `avatar_url` or does this version of the API just return it as `avatarUrl` instead of `avatar_url`?

Reply:
That's just the point, that the GraphQL query returns the keys in another notation

Reply:
FWIW I think you can make an `alias` to what you get from GQL. I'd prefer camelCase instead of snake_case though :P